doc/mgmt/SONiC_OC_Interface_HLD.md

```diff
@@ -586,6 +648,31 @@ Mode of IPv4 address assignment: MANUAL
 IPv6 address is a::b/64
 Mode of IPv6 address assignment: MANUAL
 ```
+#### Display interface COUNTERS
+`show interface counters`
+Units: RX_MBPS/TX_MBPS(MB/s), RX_MbPS|TX_MbPS(Mb/s), RX_PPS|TX_PPS(pkts/s)
+
+Interface    State    RX_OK RX_MBPS  RX_MbPS  RX_PPS  RX_UTIL    RX_ERR    RX_DRP    RX_OVR    TX_OK  TX_MBPS    TX_MbPS  TX_PPS  TX_UTIL    TX_ERR    TX_DRP    TX_OVR
```

Comment:
RX_UTILS = will it show real time packets received in % ?

Reply:
Utilization counters are calculated based on the configured load interval window. Back end calculates and populates  these counters (depends on load interval and polling interval) in counterdb and NBI just picks it up from there.
Please refer below HLD for more information:
https://github.com/BRCM-SONIC/sonic_doc_private/pull/43